R/rows.R

```diff
@@ -1,4 +1,4 @@
-#' Manipulate individual rows, optionally modifying the underlying table.
+#' Manipulate individual rows on the database
```

Comment:
I think it's really important to include "modifying the underlying" table in the title, since that makes these functions different to every other dbplyr function.